app/controllers/application_controller.rb

```diff
@@ -2170,7 +2170,7 @@ def visibility_box_edit
 
   def get_record_display_name(record)
     return record.label                      if record.respond_to?("label")
-    return record.description                if record.respond_to?("description") && !record.description.nil?
+    return record.description                if record.respond_to?("description") && record.description.nil?
```

Comment:
@Yadnyawalkya why would you return record.description if that is nil, this change is not making sense to me.